app/uk/gov/hmrc/merchandiseinbaggage/controllers/GoodsOverThresholdController.scala

```diff
@@ -48,12 +47,12 @@ class GoodsOverThresholdController @Inject()(
                   Ok(
                     view(
                       destination,
-                      totalGbpValue,
+                      calculations.totalGbpValue,
```

Comment:
why has this been addeed when we have import calculations._, need to remove the calculations. or the import statement

Reply:
because is taken from an implicit class and import calculations._ is still needed for destination & paymentCalculations